airbyte-integrations/connectors/source-amazon-seller-partner/metadata.yaml

```diff
@@ -51,6 +51,9 @@ data:
       4.0.0:
         message: Stream `GET_FBA_STORAGE_FEE_CHARGES_DATA` schema has been updated to match Amazon Seller Partner. Users will need to refresh the source schema and reset this stream after upgrading.
         upgradeDeadline: "2024-03-11"
+      5.0.0:
+        message: The PK for the stream `GET_FLAT_FILE_ALL_ORDERS_DATA_BY_ORDER_DATE_GENERAL` has been removed to prevent incorrect record deduplication in sync mode. Users will need to refresh the source schema and reset this stream after upgrading.
```

Comment:
I really like how this is explained. Can we add this reason in the migration guide as well? 

Note to @katmarkham: It seems like this is a case that wasn't documented in https://docs.google.com/spreadsheets/d/11JvsidesqJkHO8EGFr5d_T9MfkvMqSHH93Nu3tvs-zk/edit?gid=0#gid=0 so we don't have a message for this one.

Reply:
Which migration guide are you referring to?

In the amazon-seller-partner-migration.md, it’s in the first two rows.

Reply:
I'm referring to amazon-seller-partner-migration.md as the migration guide, yes. It feels relevant for the user that this was creating dedupping in the destination that would lead to missing records. This way, they might just do a count of in the source and in the destination to understand if they need to refresh

Reply:
Ah..ok. 

I've added details on how this update affects users in the migration guide.

Reply:
"has been removed to prevent incorrect record deduplication in sync mode." in what sync mode?